routes/pages.js

```diff
@@ -63,24 +63,6 @@ async function listPages (req, res, next) {
   res.status(200).json({ pages })
 }
 
-// Create new page
```

Comment:
Nit: could we also remove unused imports for this file?

Reply:
500f067